src/MediaHandler.js

```diff
@@ -359,8 +365,8 @@ export default class MediaHandler {
         return false;
       },
     };
-    if (this._auth) {
-      opts.headers.authorization = this._auth;
+    if (this._auth()) {
+      opts.headers.authorization = this._auth();
```

Comment:
hm, does the provider of that function know - if he/she passes a function instead of a string - that it will be called twice and should return the same result?

Reply:
okeeeee....

Reply:
it's anyways not correct